STANDARD.md

```diff
@@ -448,22 +448,22 @@ short-hand notation (see: *Constant Record Components*).
                               with `constant record components`
     - example: reading example (with h5py) in Python:
 ```python
-def is_const_record(record_name, component_name) :
-    return ("value" in record_name["component_name"].attrs.keys())
+def is_const_record(record, component_name) :
+    return ("value" in record[component_name].attrs.keys())
 
-def get_component(record_name, component_name) :
-    if is_const_record(record_name, component_name) :
-        return record_name["component_name"].attrs["value"]
+def get_component(record, component_name) :
+    if is_const_record(record, component_name) :
+        return record[component_name].attrs["value"]
     else :
-        record_name["component_name"][:]
+        return record[component_name].value
 
 f = h5py.File('example.h5')
 species = f["<path_to_species_group>"]
 
 position_x_relative = get_component(species["position"], "x") \
-                    * species["position"].attrs["unitSI"]
+                    * species["position/x"].attrs["unitSI"]
```

Comment:
the data method is only meant to be used on **components** (`B/x` or 'temperature', but not `B`).
`electrons/positions/x` usually returns an array of length `N` (number of particles). postpic treats every particle property like that. For `electrons/mass` this is usually a wrong assumption, but what you will get instead is an array with only a single float in it. The optimizations you are talking about (element-wise against constant) are now automatically performed in numpy and postpic doesnt have to worry about it.

Since postpic can do a lot of calculations with your data (if you want it to), not converting to float64 ended in rounding errors visible in the final result in some rare cases. I just dont want to bother and its actually a fast operation compared to reading from HDD, histogram creation or saving the fingure with matplotlib.